pubspec.yaml

```diff
@@ -5,7 +5,7 @@ publish_to: none
 
 environment:
   sdk: '>=2.18.5 <3.0.0'
-  flutter: 3.3.9
+  flutter: 3.7.0
```

Comment:
TODO for @matibat: check what are the downsides of bumping the version for localizations